src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,70 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+
+    // ADD THE PORT NUMBERS TO CONSTANTS!!!!!
+
+    public static final class DriveConstants {
+        public static final int kDriveMotor1 = 0;
+        public static final int kDriveMotor2 = 0;
+        public static final int kDriveMotor3 = 0;
+        public static final int kDriveMotor4 = 0;
+
+        public static final int kLeftTalonPort = 0;
+        public static final int kRightTalonPort = 0;
+        
+        public static final int kGyroSensor = 0; //IMU for point turn
+
+        public static final double kWheelDiameter = 0; //ADD UNIT OF MEASUREMENT
+
+    }
+
+    public static final class LauncherConstants {
+        public static final int kLauncherMotor1 = 0;
+        public static final int kLauncherMotor2 = 0;
+
+        // pistons adjust the angle of the launcher
+        public static final int kLauncherPiston1 = 0; // double solenoid
+        public static final int kLauncherPiston2 = 0; // double solenoid
+
+    }
+
+    public static final class IntakeConstants {
+        
+        // beam breaker detects the number of powercells in the cellevator
+        public static final int kBeamBreak1 = 0;
```

Comment:
There are two ports per beam break.

Reply:
We checked, and there are two ports but we only need to put the output port in the code.